gr-utils/python/modtool/cli/base.py

```diff
@@ -0,0 +1,169 @@
+#
+# Copyright 2018 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+""" Base CLI module """
+
+import os
+import sys
+import logging
+import functools
+from importlib import import_module
+from pkg_resources import iter_entry_points
+from logging import Formatter, StreamHandler
+
+import click
+from click import ClickException
+from click_plugins import with_plugins
+
+from gnuradio import gr
+
+sys.tracebacklimit = 0
```

Comment:
I thought we agreed on removing this limit and debugging the actual backtrace that was coming up